modules/metastore/src/Form/DataDictionarySettingsForm.php

```diff
@@ -119,6 +119,20 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#default_value' => $config->get('data_dictionary_sitewide'),
     ];
 
+    $form['export_titles_mode'] = [
+      '#type' => 'select',
+      '#title' => $this->t('Export Titles Mode'),
+      '#options' => [
+        'dictionary_titles' => $this->t('Use data dictionary titles'),
+        'resource_titles' => $this->t('Use the column names from the resource file'),
+      ],
+      '#default_value' => $config->get('export_titles_mode'),
+      '#description' => $this->t("Chose which title values will be used for filtered csv exports."),
```

Comment:
Nitpick, I think it should say 'Choose which title ...' and also capitalize CSV?

Reply:
🤦‍♀️ yes